io.github.hermitdemschoenenleben.linien.yml

```diff
@@ -0,0 +1,112 @@
+app-id: io.github.hermitdemschoenenleben.linien
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: launcher.sh
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --share=network
+
+modules:
+  - name: sip
+    config-opts:
+      - --no-dist-info
+      - --no-stubs
+      - --bindir=/app/bin
+      - --destdir=/app/lib/python3.7/site-packages
+      - --incdir=/app/include
+      - --pyidir=/app/lib/python3.7/site-packages
+      - --sipdir=/app/share/sip
+      - --sip-module=PyQt5.sip
+    sources:
+      - type: archive
+        url: https://www.riverbankcomputing.com/static/Downloads/sip/4.19.17/sip-4.19.17.tar.gz
+        sha256: 12bcd8f4d5feefc105bc075d12c5090ee783f7380728563c91b8b95d0ec45df3
+      - type: script
+        commands:
+          - processed=`sed -e 's|--prefix=/app||' <<< $@`
+          - python3 configure.py $processed
+        dest-filename: configure
+    cleanup:
+      - /bin
+      - /include
+
+  - name: pyqt5
+    config-opts:
+      - --assume-shared
+      - --concatenate
+      - --confirm-license
+      - --no-designer-plugin
+      - --no-dist-info
+      - --no-docstrings
+      - --no-qml-plugin
+      - --no-qsci-api
+      - --no-stubs
+      - --no-tools
+      - QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.7m/'
+      - QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python3.7m/'
+    build-options:
+      arch:
+        i386:
+          config-opts:
+            - --enable=QtCore
+            - --enable=QtGui
+            - --enable=QtNetwork
+            - --enable=QtWidgets
+            - --enable=QtOpenGL
+        x86_64:
+          config-opts:
+            - --enable=QtCore
+            - --enable=QtGui
+            - --enable=QtNetwork
+            - --enable=QtWidgets
+            - --enable=QtOpenGL
+    sources:
+      - type: archive
+        url: https://www.riverbankcomputing.com/static/Downloads/PyQt5/5.12.2/PyQt5_gpl-5.12.2.tar.gz
+        sha256: c565829e77dc9c281aa1a0cdf2eddaead4e0f844cbaf7a4408441967f03f5f0f
+      - type: script
+        commands:
+          - processed=`sed -e 's|prefix|sysroot|' <<< $@`
+          - python3 configure.py $processed
+        dest-filename: configure
+    cleanup:
+      - /share/sip
+
+  - python3-cython.json
+  - python3-wheel.json
+  - python3-numpy.json
+  - lapack.json
```

Comment:
Any reason to keep lapack/boost in separate files? Besides, you can generate modules for python dependencies with https://github.com/flatpak/flatpak-builder-tools/tree/master/pip

Reply:
no, there was no reason for this, I just changed it